modules/ROOT/pages/index.adoc

```diff
@@ -50,7 +50,7 @@ xref:release-notes.adoc#SSO-create-users[[.card-title]#Create a new user automat
 == Getting started
 [.card.card-index]
 --
-xref:getting-started-tutorial.adoc[[.card-title]#Getting started tutorial# [.card-body.card-content-overflow]#pass:q[A step-by-step creation of a first simple Living Application to learn about all the concepts in Bonita.]#]
+xref:what-is-bonita.adoc[[.card-title]#Getting started tutorial# [.card-body.card-content-overflow]#pass:q[A step-by-step creation of a first simple Living Application to learn about all the concepts in Bonita.]#]
```

Comment:
Why not keep getting-started-tutorial.adoc ?

Reply:
this page does not exist and didn't exist in the markdown format.
The original link was `[Getting started tutorial](_getting-started-tutorial.md)` (see latest commit in markdown: https://github.com/bonitasoft/bonita-doc/blob/d48c88796264584f02a65e00b56d140e178859bd/md/index.md)
All pages starting with `_` was generated from the taxonomy and the generated page was a list of links, so not very engaging.
Here, I have put the link to the 1st page of the tutorial, also available via the taxonomy.

Reply:
hm, after checking the preview, the `tutorial-overview` page seems more accurate for this link

Reply:
The getting started link will be managed by #1456, I am going to revert the change in this PR